### DISCUSSION TITLE
Livechart 2 wpf with crossing X & Y axis

### DISCUSSION THREAD

**Discussion by Rkjena10** [Live-Charts/LiveCharts2#1216] [General]
![images](https://github.com/beto-rodriguez/LiveCharts2/assets/139604399/92b51008-c3a3-4bab-8c0d-803f2b57baa2)
Hi, 
I have a simple wpf application with latest livechat.wpf package. In Cartesian chart line series I want X and Y axis crossing at center, I mean at 0 position. So that it will create a quadrilateral Cartesian, then in the middle I want to add points like above image And it will show like Cartesian plane graph. Exactly Like attached image above. Please help

**Comment by beto-rodriguez** [comment]
There is the ZeroPaint property on the axis, it enables you to set a Paint for the 0 value separator:

https://livecharts.dev/docs/WPF/2.0.0-beta.950/samples.axes.style

**Reply by Rkjena10** [reply]
Thanks, But how can i add my values there in crossing axis. Like below. 
![images](https://github.com/beto-rodriguez/LiveCharts2/assets/139604399/885a5e55-0e39-487a-9971-19fc08f58d34)
